src/rmplan/cleanup.ts

```diff
@@ -79,9 +85,9 @@ export function findYamlStart(text: string): string {
 }
 
 const doubleSlash = /\/\/\s*.*$/;
-const slashStar = /\/\*[\s\S]*?\*\//;
```

Comment:
Fix, this is from cleanup doing the wrong thing.

slashStsr needs to match end of line only, and double slash should require at least one space and then some characters